lib/markd/src/markd/renderers/html_renderer.cr

```diff
@@ -235,13 +235,8 @@ module Markd
     private def toc(node : Node)
       return unless node.type.heading?
 
-      {% if Crystal::VERSION < "1.2.0" %}
-        title = URI.encode(node.first_child.text)
-        @output_io << %(<a id="anchor-) << title << %(" class="anchor" href="#anchor-) << title << %("></a>)
-      {% else %}
-        title = URI.encode_path(node.first_child.text)
-        @output_io << %(<a id="anchor-) << title << %(" class="anchor" href="#anchor-) << title << %("></a>)
-      {% end %}
+      title = URI.encode_path(node.first_child.text)
+      @output_io << %(<a id="anchor-) << title << %(" class="anchor" href="#anchor-) << title << %("></a>)
```

Comment:
I don't think we should modify the vendored shard code.

Reply:
this isn't really relevant to this PR, is it?

Reply:
d'oh I get what you mean now, sorry